src/style.css

```
@@ -0,0 +1,112 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+}
+
+.search-bar1 {
+  padding-left: 62px;
+  height: 70px;
+  width: 100%;
+  font-weight: 300;
+  font-style: normal;
+  font-size: 16px;
+  line-height: 22px;
+  font-family: inherit;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.search-bar1:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.search-bar1:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  font-weight: 900;
+}
```

this is the same classes as the small ones, you should unify it for base class
```suggestion
.search-bar:hover {
  background: #fff;
  border: 1px solid #e1e7ed;
  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
  border-radius: 4px;
}

.search-bar:focus {
  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
  border: 1px solid #e1e7ed;
  border-radius: 3px;
  font-weight: 900;
}
```